lib/widgets/app.dart

```diff
@@ -221,15 +223,64 @@ class ChooseAccountPage extends StatelessWidget {
     required Widget title,
     Widget? subtitle,
   }) {
+    final designVariables = DesignVariables.of(context);
+    final zulipLocalizations = ZulipLocalizations.of(context);
     return Card(
       clipBehavior: Clip.hardEdge,
       child: ListTile(
         title: title,
         subtitle: subtitle,
+        trailing: PopupMenuButton<AccountItemOverflowMenuItem>(
```

Comment:
It looks like the PopupMenuButton doc recommends using the newer MenuAnchor instead. What if we used that — how would it differ?

Reply:
| `PopupMenuButton` | `MenuAnchor` |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/00479588-76a3-4077-a737-bbf40e2b2c51) | ![image](https://github.com/user-attachments/assets/e5c68e85-3755-4ba6-b77b-83b95928cc4c) |

Reply:
It's not much different, but I'm happy to use the more modern thing. 🤷‍♂️

Reply:
Yeah, SGTM